lightbulb/slash_commands/context.py

```diff
@@ -171,7 +171,9 @@ def option_values(self) -> SlashCommandOptionsWrapper:
         """The values for the slash command's various options."""
         return SlashCommandOptionsWrapper(self.options)
 
-    async def respond(self, content: hikari.UndefinedType = hikari.UNDEFINED, **kwargs: hikari.UndefinedType) -> None:
+    async def respond(
+        self, content: hikari.UndefinedType = hikari.UNDEFINED, **kwargs: hikari.UndefinedType
```

Comment:
Any reason why type-hinting these parameters to UndefinedType as opposed to `UndefinedOr[Any]` for content and `Any` for kwargs?

Reply:
Because mypy doesn't like `Any` :P
And neither can actually be `Any` and I couldn't be bothered to copy the whole signature of `create_initial_response`

Reply:
Ah well, I'll leave it as it is then.